AfricanReligions/localisation/afr_event_texts.csv

```diff
@@ -79,18 +79,33 @@ EVTDESC_MNM_3228_BORI;I walk to the center of the ceremony and make a quick deci
 EVTDESC_MNM_3228_AFRICAN;I step up to the altar and make a quick decision � as I reach the center of the stage, I collapse to the floor. The worshipers gasp as I, their [Root.GetTitle], begin quoting [Root.Religion.GetScriptureName]. I roll my eyes back as I speak, as if confronting the skies themselves. When my last words ring out, everyone seems to be holding their breath. For a moment I worry that my stunt has failed. Then the [Root.Religion.GetPriestTitle] exclaims "A VISION! My [Root.GetLordLady], let me help you stand!";;;;;;;;;;;;;x
 EVTDESC_MNM_3229_BORI;I walk to the center of the ceremony and make a quick decision � as I reach a group of people, I collapse to the floor. The worshipers gasp as I, their [Root.GetTitle], begin singing [Root.Religion.GetScriptureName]. I roll my eyes back as I sing, as if confronting the skies themselves. When my last words ring out, they start to whisper. For a moment I am convinced that my stunt has succeeded. Then the [Root.Religion.GetPriestTitle] prods at me. "Uhm, my [Root.GetLordLady]? Let's get you back onto your feet.";;;;;;;;;;;;;x
 EVTDESC_MNM_3229_AFRICAN;I step forwards and make a bold decision � as I reach a group of people, I collapse to the floor. The worshipers gasp as I, their [Root.GetTitle], begin quoting fragments of [Root.Religion.GetScriptureName]. I roll my eyes back as I speak, as if confronting the skies themselves. When my last words ring out, they start to whisper. For a moment I am convinced that my stunt has succeeded. Then the [Root.Religion.GetPriestTitle] prods at me. "Uhm, my [Root.GetLordLady]? Let's get you back onto your feet.";;;;;;;;;;;;;x
+EVTDESC_AFRREL100;Many are the religious traditions of the Western Sahel, with different names for the primordial twins, the star Sirius and even for God himself. Now I have the option to choose between one of three traditions. That of the Serer, with its complex rituals and spiritualism, that of the Dogon, masters of astronomy, or that of the Malinke, conquerors and masters of the forge.;;;;;;;;;;;;;x
```

Comment:
Think this should be ' Now I have the option to choose between one of three traditions: that of the Serer, with its complex rituals and spiritualism; that of the Dogon, masters of astronomy; or that of the Malinke, conquerors and masters of the forge.'

Reply:
You're right about the colon, but we can't use semicolons in localizations.